diff --git a/node_modules/y-monaco/src/y-monaco.js b/node_modules/y-monaco/src/y-monaco.js
index 1d768fe..df00e51 100644
--- a/node_modules/y-monaco/src/y-monaco.js
+++ b/node_modules/y-monaco/src/y-monaco.js
@@ -1,5 +1,5 @@
 import * as Y from 'yjs'
-import * as monaco from 'monaco-editor'
+import * as monaco from '@monaco-editor/react'
 import * as error from 'lib0/error'
 import { createMutex } from 'lib0/mutex'
 import { Awareness } from 'y-protocols/awareness' // eslint-disable-line
